examples/embed/plot_kcca_tutorial.py

```diff
@@ -22,10 +22,11 @@
 """
 
 
-from scipy import stats
-from mvlearn.plotting import crossviews_plot
 from mvlearn.embed import KCCA
+from mvlearn.model_selection import train_test_split
+from mvlearn.plotting import crossviews_plot
 import numpy as np
+from scipy import stats
```

Comment:
imports should go from more generic to local. So first standard lib, then numpy/scipy then mvlearn